examples/social/src/lib.rs

```diff
@@ -1,149 +1,155 @@
-//!
-//! ```ignore
-//! e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65010000000000000001000000e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65030000000000000000000000
-//! ```
-//!
-//! With the `wallet show` command you can find the ID of the application creator's chain:
-//!
-//! ```bash
-//! linera --wallet "$LINERA_WALLET1" --storage "$LINERA_STORAGE1" wallet show
-//! ```
-//!
-//! ```ignore
-//! e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65
-//! ```
-//!
-//!
-//! Now start a node service for each wallet, using two different ports:
-//!
-//! ```bash
-//! linera --wallet "$LINERA_WALLET1" --storage "$LINERA_STORAGE1" service --port 8080 &
-//! linera --wallet "$LINERA_WALLET2" --storage "$LINERA_STORAGE2" service --port 8081 &
-//! ```
-//!
-//! Point your browser to http://localhost:8081. This is the wallet that didn't create the
-//! application, so we have to request it from the creator chain. As the chain ID specify the
-//! one of the chain where it isn't registered yet:
-//!
-//! ```json
-//! mutation {
-//!     requestApplication(
-//!         chainId: "1db1936dad0717597a7743a8353c9c0191c14c3a129b258e9743aec2b4f05d03",
-//!         applicationId: "e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65010000000000000001000000e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65030000000000000000000000"
-//!     )
-//! }
-//! ```
-//!
-//! Now in both http://localhost:8080 and http://localhost:8081, this should list the
-//! application and provide a link to its GraphQL API. Remember to use each wallet's chain ID:
-//!
-//! ```json
-//! query { applications(
-//!     chainId:"1db1936dad0717597a7743a8353c9c0191c14c3a129b258e9743aec2b4f05d03"
-//! ) { id link } }
-//! ```
-//!
-//! Open both URLs under the entry `link`. Now you can use the application on each chain.
-//! E.g. in the 8081 tab subscribe to the other chain:
-//!
-//! ```json
-//! mutation {
-//!     subscribe(
-//!         chainId: "e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65"
-//!     )
-//! }
-//! ```
-//!
-//! Now make a post in the 8080 tab:
-//!
-//! ```json
-//! mutation {
-//!     post(
-//!         text: "Linera Social is the new Mastodon!"
-//!     )
-//! }
-//! ```
-//!
-//! Since 8081 is a subscriber. Let's see if it received any posts:
-//!
-//! ```json
-//! query { receivedPostsKeys { timestamp author index } }
-//! ```
-//!
-//! This should now list one entry, with timestamp, author and an index. If we view that
-//! entry, we can see the posted text:
-//!
-//! ```json
-//! query {
-//!   receivedPosts(
-//!     key: {
-//!       timestamp: 1685626618522492,
-//!       author: "e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65",
-//!       index: 0
-//!     }
-//!   )
-//! }
-//! ```
-//!
-//! ```json
-//! {
-//!   "data": {
-//!     "receivedPosts": "Linera Social is the new Mastodon!"
-//!   }
-//! }
-//! ```
+/*!
```

Comment:
I tend to use block comments for large blocks of documentation since it makes it easier to edit — if it's against Linera policy let me know and I can switch it back :)